rpcs3/Emu/Cell/Modules/cellSearch.cpp

```diff
@@ -280,12 +331,14 @@ error_code cellSearchStartListSearch(CellSearchListSearchType type, CellSearchSo
 					std::shared_ptr<search_content_t> curr_find = std::make_shared<search_content_t>();
 					if (item_path.length() > CELL_SEARCH_PATH_LEN_MAX)
 					{
-						// Create mapping which will be resolved to an actual hard link in VFS by cellSearchPrepareFile
-						std::string link = "/.tmp/" + std::to_string(hash) + item.name.substr(ext_offset);
-						strcpy_trunc(curr_find->infoPath.contentPath, link);
+						// TODO: Create mapping which will be resolved to an actual hard link in VFS by cellSearchPrepareFile
+						cellSearch.warning("cellSearchStartListSearch(): Music-Directory Name \"%s\" is too long: %i", item_path, item_path.length());
```

Comment:
Why are you removing the code here?
Also, the warning is wrong. this can be anything, it doesn't have to be a Music-Directory

Reply:
In my testing, this code always produced the following error that I could not get rid of:

`·E 0:00:14.398563 {RSX [0x0074128]} RSX: nv406e::semaphore_acquire has timed out. semaphore_address=0x50300490`

I wanted to postpone this problem until I have more time to analyze this problem.